File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/metadata/read/MetadataReadResponseTest.java
package uk.ac.jisc.rdss.tests.metadata.read;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class MetadataReadResponseTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/metadata/read/response_schema.json";
    }

    @Test
    public void validateResponse() throws Exception {
        validateJson("messages/metadata/read/response.json");
    }
}

File: message-api-schema-validator/tests/metadata/metadata_read_response_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class MetadataReadResponseTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/metadata/read/response_schema.json"

    def test_validate_request(self):
        self.validate_json("messages/metadata/read/response.json")
